scripts/minikube.sh

```diff
@@ -202,6 +211,14 @@ up)
 
     disable_storage_addons
 
+    #  get kubernetes version we are operating on and accordingly enable feature gates
+    KUBE_MAJOR=$(kube_version 1)
+    KUBE_MINOR=$(kube_version 2)
+    if  [ "${KUBE_MAJOR}" -eq 1 ] && [ "${KUBE_MINOR}" -ge 20 ]
+    then
+    # if kubernetes version is greater than 1.20 enable Generic Ephemeral Volume feature gate
+    K8S_FEATURE_GATES="${K8S_FEATURE_GATES},GenericEphemeralVolume=true"
```

Comment:
if `K8S_FEATURE_GATES` is an empty string, maybe the extra `,` can cause issues?

Reply:
as discussed, its not causing issue rather its going through, so will keep this version.